extensions/amp-form/0.1/amp-form.js

```diff
@@ -974,33 +953,25 @@ export class AmpForm {
     }
     return promise.then(responseJson => {
       this.triggerFormSubmitInAnalytics_('amp-form-submit-error');
-      this.handleSubmitFailure_(
-        e,
-        responseJson,
-        /** getSsrEventResponse */ false
-      );
+      this.handleSubmitFailure_(e, responseJson, responseJson);
```

Comment:
I am not an expert on styling but I do feel confused about this API where two same objects are passed in most of the time... But I'll let you folks to decide.

Reply:
I think it's ok, though we add inline param comments or just make it optional:

```js
this.handleSubmitSuccess_(e, responseJson);

handleSubmitSuccess_(error, json, opt_eventData) {
  ...
  this.triggerAction_(FormEvents.SUBMIT_SUCCESS, opt_eventData || json);
}
```

Reply:
+1 on making it optional

Reply:
If we want to propagate `null` from the response.body, then `opt_eventData || json` would still return the full response from the SSR, no?

Reply:
If body can be falsy then you can do `opt_eventData === undefined ? json : opt_eventData`.